music/.gitignore

```diff
@@ -5,3 +5,5 @@ es6/
 node/
 dist/
 yarn-error.log
+report.*.json
+tsconfig.json
```

Comment:
why is the tsconfig being ignored? seems bad

Reply:
afaik, vscode only takes tsconfig.json as the config but since this code repository only has the es5 and 6 flavors, i didn't want to include another tsconfig.json for myself just to see errors inline in my editor.

https://code.visualstudio.com/docs/typescript/typescript-compiling#_tsconfigjson

Reply:
prob shd also link to the issue: https://github.com/Microsoft/vscode/issues/12463